test/Microsoft.Azure.SignalR.Common.Tests/Auth/AzureActiveDirectoryTests.cs

```diff
@@ -17,42 +19,24 @@ public class AzureActiveDirectoryTests
         private const string TestClientSecret = "";
         private const string TestTenantId = "";
 
-        [Fact]
-        public void TestBuildAuthority()
-        {
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", ""));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", "An invalid tenant id"));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", ".00000000-0000-0000-0000-000000000000"));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", "00000000-0000-0000-0000-000000000000."));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", "00000000-0000-0000-θθθθ-000000000000"));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", "0000-0000-0000-θθθθθθθθ-000000000000"));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", "00000000-abcd-efgh-0000-000000000000"));
-            Assert.Throws<FormatException>(() => new AadApplicationOptions("foo", "00000000-ABCD-EFGH-0000-000000000000"));
-
-            AadApplicationOptions options;
-
-            options = new AadApplicationOptions("foo", "00000000-0000-0000-0000-000000000000");
-            Assert.Equal("https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000", options.BuildAuthority().ToString());
-
-            options = new AadApplicationOptions("foo", "00000000-abcd-efab-cdef-000000000000");
-            options.WithDogfood();
-            Assert.Equal("https://login.windows-ppe.net/00000000-abcd-efab-cdef-000000000000", options.BuildAuthority().ToString());
-        }
+        private static readonly string[] DefaultScopes = new string[] { "https://signalr.azure.com/.default" };
 
         [Fact(Skip = "Provide valid aad options")]
```

Comment:
Seems better put into e2e test

Reply:
We did have a nightly test for this, this is only for dev usage.